tests/source/comments-in-lists/wrap-comments-not-normalized.rs

```diff
@@ -13,9 +13,8 @@ pub enum E {
 }
 
 pub enum E2 {
-    // This can be changed once https://github.com/rust-lang/rustfmt/issues/4854 is fixed
-// Expand as needed, numbers should be ascending according to the stage
-// through the inclusion pipeline, or according to the descriptions
+    // Expand as needed, numbers should be ascending according to the stage
+    // through the inclusion pipeline, or according to the descriptions
```

Comment:
Was it necessary to modify these files? It's typically a flag when we have to change existing inputs to tests, but I'm guessing this isn't actually necessary, perhaps wasn't even explicitly intentional?

Reply:
Ah sorry, I just thought we might want to remove the `// This can be changed once https://github.com/rust-lang/rustfmt/issues/4854 is fixed` part of the comment as the issue should be addressed with the PR. It makes sense to leave it as it is tho, so I will revert that change.

Reply:
@calebcartwright are you trying to highlight that this file is the ``source`` file and that we don't need to update the formatting since it'll get checked against the corresponding ``target``? Would we not want to remove the comment about issue 4854 to not confuse anyone down the line about whether or not the issue is fixed?

Reply:
I was really just speaking to changing of the indentation, I actually hadn't read the the comment text word for word. I'm okay with removing that first line though would prefer to defer that to a follow up PR if anyone is interested given how long this PR (and it's predecessor) have been basically ready, but would still be good in that potential follow up to leave the `source` contents with incorrect indentation